src/pages/base-page.ts

```diff
@@ -51,6 +56,10 @@ export class BasePage {
 
 
     // Search list of elements
+    public allBySystelabTestId(dataTestId: string): ElementArrayFinder {
+        return this.current.allByCSS(`[${Test.SYSTELAB_TEST_ID_ATTRIBUTE}="${dataTestId}"]`);
```

Comment:
Why not calling from here to the new method added at ElementFinder level? allBySystelabTestId()

Reply:
Applied (also to bySystelabTestId)